src/tribler/core/components/libtorrent/tests/conftest.py

```diff
@@ -15,10 +17,10 @@ def tdef():
 
 
 @pytest.fixture
-def mock_download_config(mocker, test_download):
-    return mocker.patch.object(test_download, 'config')
+def mock_download_config(test_download):
+    test_download.config = MagicMock()
```

Comment:
It looks like this fixture is used in two tests only; can we inline this logic into the test itself and remove the fixture?